Readme.md

```diff
@@ -110,9 +115,9 @@ cons.swig('views/page.html', { user: 'tobi' })
 
  To enable caching simply pass `{ cache: true }`. Engines _may_ use this option to cache things reading the file contents, compiled `Function`s etc. Engines which do _not_ support this may simply ignore it. All engines that consolidate.js implements I/O for will cache the file contents, ideal for production environments.
  When using consolidate directly: `cons.swig('views/page.html', { user: 'tobi', cache:true }, callback);`
- Using Express 3 or higher: `app.locals.cache = true` or set NODE_ENV to 'production' and Express will do this for you.
+ Using Express or higher: `app.locals.cache = true` or set NODE_ENV to 'production' and Express will do this for you.
```

Comment:
I think this should say `Using support Express versions:` instead of `Using Express or higher:`